### ISSUE TITLE
accessible type should be called tagName instead

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#167]
It would be more conventional, type is overloaded with input type.

**Comment by jessegreenberg** [comment]
This was handled via #166.  Closing.
